packages/supermassive/src/executeWithoutSchema.ts

```diff
@@ -626,6 +626,10 @@ function executeSubscriptionImpl(
   const { groupedFieldSet } = collectFields(exeContext, rootTypeName);
 
   const firstRootField = groupedFieldSet.entries().next().value;
+  if (firstRootField === undefined) {
```

Comment:
note: For ts 5.5 it is still any type but ts 5.6 has proper types and this type guard will be needed.